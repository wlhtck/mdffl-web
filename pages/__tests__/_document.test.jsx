import React from 'react';
import { shallow } from 'enzyme';
import Document from '../_document';

describe('pages/index', () => {
  it('renders an index page', () => {
    expect(shallow(<Document />)).toMatchSnapshot();
  });
});
