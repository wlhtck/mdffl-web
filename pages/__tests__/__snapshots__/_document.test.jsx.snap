// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index renders an the Document 1`] = `
<html
  lang="en-US"
>
  <Head>
    <link
      href="https://fonts.googleapis.com/css?family=Lato:400,700|Raleway:600,700,800"
      rel="stylesheet"
    />
    <title>
      My Page Title
    </title>
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</html>
`;
